### ISSUE TITLE
Can't launch the game

### ISSUE THREAD

**Issue by K0-RR** [herbix/chunkLauncher#10]
I selected MS account as auth type and entered my email in the username field. I also tried to use the offline mode but had no luck. Every time I click the Launch button it doesn't do anything or it opens a new card in a browser that says "Are you trying to sign in to ChunkLauncher?" I accept this and I get the information that I can now close this window but after clicking the Launch button again the same URL is opened.

**Comment by herbix** [comment]
Did you see any error message above progress bar before clicking Launch button again?

**Comment by K0-RR** [comment]
Nope, just tried again. Nothing when launching from the terminal too besides this when it opened browser tab with MS login
```
Opening in existing browser session.
[487575:487575:0100/000000.785919:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
```

**Comment by herbix** [comment]
Could you try running it in command line and paste stdout here?
```sh
java -jar ChunkLauncher.jar
```

**Comment by herbix** [comment]
I tried ChunkLauncher on ubuntu, it didn't work in many aspect. I'll investigate more about it.
